### PR TITLE
removing misc deprecations, which were missed earlier

### PR DESCRIPTION
**Pull request by gsps1**

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ApplicationLifecycleService.java [hunk 1]
```diff
@@ -718,8 +718,8 @@ private void deleteApp(final ApplicationId appId, ApplicationSpecification spec)
     // Delete all streams and queues state of each flow
     // TODO: This should be unified with the DeletedProgramHandlerStage
     for (final FlowSpecification flowSpecification : spec.getFlows().values()) {
-      Id.Program flowProgramId = Id.Program.from(idApplication, ProgramType.FLOW, flowSpecification.getName());
-
+      ProgramId flowProgramId = new ProgramId(idApplication.getNamespaceId(), idApplication.getId(),
```
Comment (ecfm) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ApplicationLifecycleService.java]: why still keep the `Id.Application`? Can also use `appId.program(ProgramType.FLOW, flowSpecification.getName())`

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ApplicationLifecycleService.java [hunk 2]
```diff
@@ -729,7 +729,7 @@ private void deleteApp(final ApplicationId appId, ApplicationSpecification spec)
         }
       }
       // Remove all process states and group states for each stream
-      final String namespace = String.format("%s.%s", flowProgramId.getApplicationId(), flowProgramId.getId());
+      final String namespace = String.format("%s.%s", flowProgramId.getApplication(), flowProgramId.getEntityName());
```
Comment (ecfm) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ApplicationLifecycleService.java]: it's clearer to use `flowProgramId.getProgram()`

**Comment by ecfm** [comment]
LGTM

**Comment by gsps1** [comment]
build green https://builds.cask.co/browse/CDAP-DUT6201-1

**Pull request merged**